rudder-core/src/main/scala/com/normation/rudder/domain/nodes/NodeInfo.scala

```diff
@@ -62,13 +60,21 @@ case class NodeInfo(
   , agentsName    : Seq[AgentType]
   , policyServerId: NodeId
   , localAdministratorAccountName: String
-  , creationDate  : DateTime
-  , isBroken      : Boolean
-  , isSystem      : Boolean
-  , isPolicyServer: Boolean
   //for now, isPolicyServer and server role ARE NOT
   //dependant. So EXPECTS inconsistencies.
   //TODO: remove isPolicyServer, and pattern match on
   //      on role everywhere.
   , serverRoles   : Set[ServerRole]
-) extends HashcodeCaching
+) extends HashcodeCaching {
+
+  val id                         = node.id
+  val name                       = node.name
+  val description                = node.description
+  //val creationDate               = node.creationDate
```

Comment:
Why do you remove creationDate, nodeReportingConfiguration and properties which are not in Node ?